wixcode-users/wix-users.service.json

```diff
@@ -846,10 +953,161 @@
+              [ "The `setConsentPolicy()` function returns a Promise that resolves to the",
+                "consent policy details of the user.",
+                "",
+                "You can use the [`onConsentPolicyChanged()`](#onConsentPolicyChanged) event to listen for ",
+                "changes made when a visitor changes their consent policy with `setConsentPolicy()`. Handle ",
+                "the policy change accordingly in the event handler's [ConsentPolicyChangedHandler callback function](#ConsentPolicyChangedHandler). ",
+                "Changes to the consent policy take affect after the page is refreshed.  ",
+                "",
+                "The consent policy helps you comply with [GDPR](https://support.wix.com/en/article/preparing-your-wix-site-for-the-gdpr) and [CCPA](https://support.wix.com/en/article/preparing-your-wix-site-for-ccpa) regulations and lets your users ",
+                "control their information. The site owner sets up the site consent policy and cookie consent banner in ",
+                "the site's [Privacy Center](https://www.wix.com/my-account/site-selector/?buttonText=Open%20Privacy%20Center&title=Select%20a%20Site&autoSelectOnSingleSite=true&actionUrl=https:%2F%2Fwww.wix.com%2Fdashboard%2F%7B%7BmetaSiteId%7D%7D%2Fcookie-consent-settings-app?referralInfo=corvid-docs).",
+                " ",
+                "> **Notes:**",
+                "> + The APIs in `wix-users` are only partially functional when previewing your site.",
+                "  View a published version of your site to see their complete functionality.",
+                ">",
+                "> + The APIs in `wix-users` can only be used once the page has loaded. Therefore,",
+                " you must use them in code that is contained in or is called from the",
+                " [onReady()]($w.html#onReady) event handler or any element event handler.",
+                "> ",
+                "> + Cookie consent banners are only available on the published version of your site." ],
+            "links": [],
+            "examples":
+              [ { "title": "Set a user's consent policy",
+                  "body":
+                    [ "import { setConsentPolicy } from 'wix-users';",
+                      "",
+                      "$w.onReady(function () {",
+                      "\tlet myPolicy = {",
+                      "\t\t\"defaultPolicy\": false,",
+                      "\t\t\"policy\": {",
+                      "\t\t\t\"essential\": true,",
+                      "\t\t\t\"analytics\": false,",
+                      "\t\t\t\"functional\": false,",
+                      "\t\t\t\"advertising\": false",
+                      "\t\t},",
+                      "\t\t\"createdDate\": new Date()",
+                      "\t};",
+                      "\tsetConsentPolicy(myPolicy)",
+                      "\t\t.then((policyDetails) => {",
+                      "\t\t\tconsole.log(policyDetails);",
```

Comment:
We usually assign some part of the return object to a variable instead of doing a console.log. We also add a catch to .thens and console log the error. And then show the full object below like you did.